includes/wikia/transaction/TransactionTraceNewrelic.php

```diff
@@ -38,7 +38,8 @@ function __construct() {
 
 		// report request ID as a custom request parameter for easier debugging of slow transactions
 		if ( function_exists( 'newrelic_add_custom_parameter' ) ) {
-			newrelic_add_custom_parameter( 'requestId', RequestId::instance()->getRequestId() );
+			newrelic_add_custom_parameter( 'requestId', WikiaTracer::instance()->getTraceId() );
```

Comment:
Do we want to keep both custom NewRelic params here? Or is `requestId` going to stay here just for a while?